azurerm/validators_test.go

```diff
@@ -182,3 +182,37 @@ func TestValidateIntBetweenDivisibleBy(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateAzureVMTimeZone(t *testing.T) {
```

Comment:
`VM` -> `VirtualMachine` (we spell out the resource name for tests)

Reply:
Fixed.